README.md

```diff
@@ -44,65 +45,106 @@ Usage instructions are available in the bootstrap [README](./0-bootstrap/README.
 
+#### DNS Hub
+
+Under the common folder, one project is created. This project will host the DNS Hub for the organization.
+
+#### Interconnect
+
+Under the common folder, one project is created. This project will host the Interconnect infrastructure for the organization.
+
+#### SCC Notification
+
+Under the common folder, one project is created. This project will host the SCC Notification infra structure at organization level.
+This project will contain a Pub/Sub topic and subscription, a SCC Notification configured to send all new Findings to the topic created.
+
+
+#### Secrets
+
+Under the common folder, one project is created. This project will host the Secret Manager for secrets shared by the organization.
+
+Usage instructions are available for the org step in the [README](./1-org/README.md).
+
+### [2. environments](./2-environments/)
+
+The purpose of this stage is to set up the environments folders used to house projects which contain monitoring, secrets, networking projects.
+This will create the following folder & project structure:
+
+```
+example-organization
+└── dev
+    ├── prj-d-monitoring
+    ├── prj-d-secrets
+    ├── prj-d-shared-base
+    └── prj-d-shared-restricted
+└── nonprod
+    ├── prj-n-monitoring
+    ├── prj-n-secrets
+    ├── prj-n-shared-base
+    └── prj-n-shared-restricted
+└── prod
+    ├── prj-p-monitoring
+    ├── prj-p-secrets
+    ├── prj-p-shared-base
+    └── prj-p-shared-restricted
+```
+
 #### Monitoring
 
-Under the monitoring folder, a project is created per environment (prod & nonprod) which is intended to be used as a [Cloud Monitoring workspace](https://cloud.google.com/monitoring/workspaces) for all projects in that environment.
+Under the environment folder, a project is created per environment (dev, prod & nonprod) which is intended to be used as a [Cloud Monitoring workspace](https://cloud.google.com/monitoring/workspaces) for all projects in that environment.
 Please note that creating the [workspace and linking projects](https://cloud.google.com/monitoring/workspaces/create) can currently only be completed through the Cloud Console.
 If you have strong IAM requirements for these monitoring workspaces, it is worth considering creating these at a more granular level such as per business unit or per application.
 
 #### Networking
 
-Under the networking folder, a project is created per environment (prod & nonprod) which is intended to be used as a [Shared VPC Host project](https://cloud.google.com/vpc/docs/shared-vpc) for all projects in that environment.
+Under the environment folder, a project is created per environment (dev, prod & nonprod) which is intended to be used as a [Shared VPC Host project](https://cloud.google.com/vpc/docs/shared-vpc) for all projects in that environment.
```

Comment:
It might be worth mentioning there are two projects, one for restricted and the other for private.